python/seldon_core/metrics.py

```diff
@@ -1,10 +1,115 @@
+from prometheus_client.core import (
+    HistogramMetricFamily,
+    GaugeMetricFamily,
+    CounterMetricFamily,
+    CollectorRegistry,
+)
+from prometheus_client import exposition
+from prometheus_client.utils import floatToGoString
+
+from multiprocessing import Manager
+
+import numpy as np
+
 from typing import List, Dict
+import logging
+import os
+
+
+logger = logging.getLogger(__name__)
+
 
 COUNTER = "COUNTER"
 GAUGE = "GAUGE"
 TIMER = "TIMER"
 
 
+def generate_metrics(metrics):
+    myregistry = CollectorRegistry()
+    myregistry.register(metrics)
+    return (
+        exposition.generate_latest(myregistry).decode("utf-8"),
+        exposition.CONTENT_TYPE_LATEST,
+    )
+
+
+BINS = [0] + list(np.logspace(-3, np.log10(30), 50)) + [np.inf]
+LABELS = ["worker-id", "model", "image"]
+
+my_labels = {"model": "latest", "image": "my-image"}
```

Comment:
I assume this is a placeholder for now, is this for backwards compatibility? If so maybe let's discuss around how we could do this by adding these variables as env, or alternatively to add something that is more ovious such as "NOT_IMPLEMENTED SEE #1507" - where the number would be the issue that will address this

Reply:
That was only a placeholder during my dev now. I will be adding reading those from the environmental variables (and seeing what is missing there) today.